handlers.go

```diff
@@ -120,3 +143,20 @@ func deleteUser(w http.ResponseWriter, r *http.Request) {
 		}
 	}
 }
+
+func getCookieHandler(w http.ResponseWriter, r *http.Request, cookie_name string) int {
+	value, err := cookies.Read(r, cookie_name)
+	if err != nil {
+		switch {
+		case errors.Is(err, http.ErrNoCookie):
+			http.Error(w, "cookie not found", http.StatusBadRequest)
+		case errors.Is(err, cookies.ErrInvalidValue):
+			http.Error(w, "invalid cookie", http.StatusBadRequest)
+		default:
+			log.Println(err)
+			http.Error(w, "server error", http.StatusInternalServerError)
+		}
+		return 0
+	}
+	return strconv.Atoi(value)
```

Comment:
@seoyoungcho213 Handle Atoi's returned error

Reply:
you could also simply make the function return int, err and the error would get passed along.